content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```
@@ -265,4 +318,13 @@ def error(type)
   exit MIQ_OK
 end
 
+@reconfigure_request = @miq_request.type == "VmReconfigureRequest"
+if @reconfigure_request
+  @check_quota = false # default, unless additional quota is requested
+  vm_id = @miq_request.options[:src_ids]
+  @vm = $evm.vmdb(:vm).find_by(:id => vm_id)
```

@billfitzgerald0120 
Can we raise an error if the vm is not found.